src/pages/student/quizzes/QuestionList/QuestionAnswer/components/FillInTheBlankType/index.js

```diff
@@ -12,31 +13,26 @@ const FillInTheBlankType = ({ question, getAnswer, answer, getPoint }) => {
   return (
     <Fragment>
       <div className={style.question}>
-        <p className={style.paragraph}>{`${question.question}`}</p>
-        {/* <p className={style.paragraph}>Note: Do not use acronyms</p> */}
-        {/* <img
-          className={style.sizeOfAvatarInQuestion}
-          alt='avatar'
-          src='https://freeiconshop.com/wp-content/uploads/edd/image-solid.png'
-        /> */}
-        <input
-          className={style.cardbody2}
-          type="text"
-          name="answer"
-          value={answer}
-          placeholder="Type your answer here"
-          onChange={(e) => {
-            if (e.nativeEvent.data != ' ') {
-              if (e.target.value === question.text_answer) {
-                setPoint(1);
-              } else {
-                setPoint(0);
+        <p>{`${question.question}`}</p>
```

Comment:
i think no need for string interpolation since we are displaying only the variable
```suggestion
        <p>{question.question}</p>
```